### PR TITLE
fix(platform,core): update styles lib version

### PR DESCRIPTION
**Pull request by N1XUS**
#### Please provide a link to the associated issue.
Closes https://github.com/SAP/fundamental-ngx/issues/5855

#### Please provide a brief summary of this pull request.
Update styles lib version in package to fix error with invalid selector.
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [n/a] tests for the changes that have been done
- [n/a] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [n/a] update `README.md`
- [n/a] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [n/a] Documentation Examples
- [n/a] Stackblitz works for all examples

### CODE REVIEW

**Review by Smiranin — APPROVED** [review]

**Review by dimamarksman — APPROVED** [review]

**Review by sKudum — APPROVED** [review]

**Comment by sKudum** [comment]
Hi @N1XUS ,

Other than build issues rest looks good.

**Review by droshev — APPROVED** [review]

**Review by aberikashvili — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -113,7 +113,7 @@
     "classlist.js": "1.1.20150312",
     "core-js": "3.7.0",
     "focus-trap": "6.5.0",
-    "fundamental-styles": "0.18.1-rc.25",
+    "fundamental-styles": "0.19.0-rc.23",
```
Comment (droshev) [inline package.json:116]: Actually, are we sure we want to bring that version without adopting the changes?

Comment (N1XUS) [reply package.json:116]: We need a newer version with fixed selectors, otherwise unit tests will fail while searching for the invalid element

**Pull request merged**